src/intercooler.js

```diff
@@ -474,6 +474,7 @@ var Intercooler = Intercooler || (function() {
         }
 
         var beforeHeaders = new Date();
+        var oldTitle = document.title;
```

Comment:
This variable seems a little lonely here, since it's only needed a few levels deep in the following code. Come to think of it, `snapshotForHistory()` could just as well access `document.title` directly itself, instead of having the old title passed in, which would be consistent with how it already accesses the old HTML and Y-offset.

Reply:
This will not work. Please notice that we call `processHeaders` just a few lines below. Inside `processHeaders` we will change the document's title. That is why we need to store the old title before we enter `processHeaders`.

Reply:
Got it, thanks!